src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int myResult = 0;
+        myResult = ((result1 > result2) && (bet1 > bet2)
+                || (result1 < result2) && (bet1 < bet2)
+                || (result1 == result2) && (bet1 == bet2)) ? 1 : 0;
+        myResult = ((result1 == bet1) && (result2 == bet2)) ? 2 : myResult;
+        return myResult;
```

Comment:
cool, but we can simplify this code
```suggestion
        int myResult = ((result1 > result2) && (bet1 > bet2)
                || (result1 < result2) && (bet1 < bet2)
                || (result1 == result2) && (bet1 == bet2)) ? 1 : 0;
        return ((result1 == bet1) && (result2 == bet2)) ? 2 : myResult;
```

Reply:
I though my previous version is easier to read (small steps in a time), but your suggestion is nicer.